SKAToolKit/SKATiledMap/SKATiledMap.m

```diff
@@ -772,20 +762,31 @@ - (void)cullAroundIndexX:(NSInteger)x
             endingY = 0;
         }
 
+        NSMutableArray *spritesToTrash = [[NSMutableArray alloc] init]; //array to hold sprites that have moved off-screen
+        [self.visibleArray
+            enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
```

Comment:
We can improve that in a separate pull request. I'll merge this into development and it'll be in our next release. Thanks @sidmani!